src/Adapter/Presenter/Order/OrderLazyArray.php

```diff
@@ -238,12 +238,22 @@ public function getAmounts()
             'value' => $this->priceFormatter->format($amount, Currency::getCurrencyInstance((int) $order->id_currency)),
         );
 
-        $amounts['totals']['total_paid'] = array(
```

Comment:
I dont think we should remove `$amounts['totals']['total_paid']`, only add `$amounts['totals']['total_including_tax']` and `$amounts['totals']['total_excluding_tax']`

Reply:
I put it back
Thank you ;)